1.1+8.16 does not compile on 32/64 bit Windows

I added MetaCoq to Coq Platform as requested here (https://github.com/coq/platform/issues/292)

Coq Platform also has a 32 bit Windows build. MetaCoq does note build here:
```
366	CAMLOPT -c -for-pack Metacoq_template_plugin gen-src/cRelationClasses.ml
   367	CAMLOPT -c -for-pack Metacoq_template_plugin gen-src/compare_dec.ml
   368	File "gen-src/cRelationClasses.ml", line 1:
   369	Error: The implementation gen-src/cRelationClasses.ml
   370	       does not match the interface gen-src\cRelationClasses.cmi: 
   371	       Values do not match:
   372	         val eq_equivalence : ('_weak1, '_weak2) coq_Equivalence
   373	       is not included in
   374	         val eq_equivalence : ('a1, __) coq_Equivalence
   375	       File "gen-src/cRelationClasses.mli", line 106, characters 0-46:
   376	         Expected declaration
   377	       File "gen-src/cRelationClasses.ml", line 211, characters 4-18:
   378	         Actual declaration
   379	make[3]: *** [Makefile.plugin:741: gen-src/cRelationClasses.cmx] Error 2
   380	make[3]: *** Waiting for unfinished jobs....
   381	make[2]: *** [Makefile.plugin:409: all] Error 2
   382	make[2]: Leaving directory '/home/runneradmin/.opam/__coq-platform.2022.09.0~8.16~2022.09~beta1/.opam-switch/build/coq-metacoq-template.1.1+8.16/template-coq'
   383	make[1]: *** [Makefile:17: plugin] Error 2
   384	make[1]: Leaving directory '/home/runneradmin/.opam/__coq-platform.2022.09.0~8.16~2022.09~beta1/.opam-switch/build/coq-metacoq-template.1.1+8.16/template-coq'
   385	make: *** [Makefile:93: template-coq] Error 2
```
See [CI run](https://github.com/coq/platform/actions/runs/3152865197/jobs/5128625185#step:5:2789).

--- a/template-coq/update_plugin.sh
+++ b/template-coq/update_plugin.sh
@@ -12,6 +12,12 @@ then
     echo "Renaming files to camelCase"
     (cd gen-src; ./to-lower.sh)
     rm -f gen-src/*.d gen-src/*.cm*
+    echo "Prepare line endings for patching (for Windows)"
+    for f in gen-src/*.ml*
+    do
+      tr -d '\r' < "$f" > tmp
+      mv -f tmp $f
+    done
     # Fix an extraction bug: wrong type annotation on eq_equivalence
     patch -N -p0 < extraction.patch
     patch -N -p0 < specFloat.patch

